apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -142,15 +142,15 @@ const BookingPage = (props: BookingPageProps) => {
     [LocationType.Huddle01]: "Huddle01 Video",
     [LocationType.Tandem]: "Tandem Video",
   };
-
+  const loggedInIsOwner = eventType.users[0].name === session?.user.name;
   const defaultValues = () => {
     if (!rescheduleUid) {
       return {
-        name: (router.query.name as string) || "",
-        email: (router.query.email as string) || "",
+        name: loggedInIsOwner ? "" : session?.user?.name || (router.query.name as string) || "",
+        email: loggedInIsOwner ? "" : session?.user?.email || (router.query.email as string) || "",
```

Comment:
This is the important bit in the defaultValues(), where we assign session.user.name and email to each input, in the case of the user not being the same as logged in